lib/game.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require_relative 'board.rb'
+require_relative 'player.rb'
+# Game class
+class Game
+  include UI
+
+  attr_accessor :continue
+  def initialize(board, player1, player2)
+    @new_board = board
+    @player1 = player1
+    @player2 = player2
+  end
+
+  def play
+    until game_is_over
+      @player1.make_move(@new_board) unless has_won?(@player2) || !@new_board.not_full?
+      @player2.make_move(@new_board) unless has_won?(@player1) || !@new_board.not_full?
+      display_message if game_is_over
+    end
+  end
+
+  private
+
+  def display_message
```

Comment:
You did great here separating this method. Now let's go further by transferring this over into the UI module. While it's already a better version than having all the `puts` and `print`, still it appears that it's not `Game` 's responsibility to hold the logic for displaying a message. But `Game` obviously can use a module's display method :)